client/src/components/ratingsReviews/RatingsAndReviews.jsx

```diff
@@ -36,14 +41,16 @@ class RatingsAndReviews extends React.Component {
     this.getDefaultReviews = this.getDefaultReviews.bind(this);
     this.handleReviewSort = this.handleReviewSort.bind(this);
     this.handleRatingProgressFilter = this.handleRatingProgressFilter.bind(this);
+    this.loadMoreReviews = this.loadMoreReviews.bind(this);
   }
 
-  getReviews(sort, id) {
+  getReviews(sort, count, id) {
     console.log('Getting Reviews');
-    axios.get(`reviews?sort=${sort}&product_id=${id}`)
+    axios.get(`reviews?sort=${sort}&count=100&product_id=${id}`)
       .then(res => {
         console.log('AXIOS GET REVIEWS:', res);
         this.reviews = res.data.results;
+        this.displayedReviews = res.data.results.slice();
```

Comment:
I just wonder, is this.reviews and this.displayedReviews is equal

Reply:
Yes, this.reviews is being referenced in this.state.reviews, while this.displayedReviews starts off with 2 and receives 2 more on button click. I decided to separate them in case i need a separate reference to all reviews on a global level